pastis/io/read/hiclib.py

```diff
@@ -0,0 +1,101 @@
+import warnings
+import numpy as np
+from scipy import sparse
+import pandas as pd
+
+
+def load_hiclib_lengths(filename):
```

Comment:
I think this is exactly the same function as the iced load_lengths function. We should be using the iced one and not duplicate code between the two libraries.

Reply:
Just fixed this, thanks for pointing it out!